### PR TITLE
[PS-1419] update productcat lib with mps 

### PR DESCRIPTION
**Pull request by swVisenze**
1. add mps api 
2. change the default resize to 1024

### CODE REVIEW

File: productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java [hunk 1]
```diff
@@ -120,6 +122,73 @@ public static ResultList parseResult(String jsonResponse, String type) {
...
+                    JSONObject jsonItem = objArray.getJSONObject(i);
+
+                    if (jsonItem.has(TYPE)) {
+                        obj.setType((jsonItem.getString(TYPE)));
+                    }
+
+                    if (jsonItem.has(BOX)) {
+
+                        JSONArray boxCoordinate = jsonItem.getJSONArray(BOX);
+                        Box box = new Box(boxCoordinate.getInt(0),
```
Comment (thehung111) [inline productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java:147]: box might be empty, should add a check here e.g. if our API is called with `box` parameter, the box would be empty array.

File: productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java [hunk 2]
```diff
@@ -120,6 +122,73 @@ public static ResultList parseResult(String jsonResponse, String type) {
...
+
+                    ObjectList obj = new ObjectList();
+                    if (jsonObject.has(REQID)) {
+                        obj.setReqid(jsonObject.getString(REQID));
+                    }
+
+                    JSONObject jsonItem = objArray.getJSONObject(i);
+
+                    if (jsonItem.has(TYPE)) {
+                        obj.setType((jsonItem.getString(TYPE)));
```
Comment (thehung111) [inline productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java:141]: next time can also use optString (then no need has method)

**Review by thehung111 — APPROVED** [review]
LGTM

File: productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java (hunk 1 above)
Comment (swVisenze) [reply productcat/src/main/java/com/visenze/productcat/android/util/ResponseParser.java]: updated already

**Review by thehung111 — APPROVED** [review]

**Pull request merged**